src/utilities.jl

```diff
@@ -0,0 +1,79 @@
+"""
+Get the latest VersionNumber for base_path/Versions.toml
+"""
+function _get_latest_version(base_path::AbstractString)
+    versions_file_path = joinpath(base_path, "Versions.toml")
+
+    if isfile(versions_file_path)
+        versions_content = parsefile(versions_file_path)
+        versions = [VersionNumber(v) for v in collect(keys(versions_content))]
+
+        return first(findmax(versions))
+    end
+end
+
+
+"""
+Use levenshtein distance to find packages closely named to pkg_to_compare
+"""
+function _find_alternative_packages(pkg_to_compare::String, packages::Array)
+    pkg_to_compare = uppercase(pkg_to_compare)
+    pkg_distances = [(REPL.levenshtein(uppercase(pkg), pkg_to_compare), pkg) for pkg in packages]
+    sort!(pkg_distances)
+    counts = [count(x -> x[1] <= i, pkg_distances) for i in 0:2]
+    max_distance = max(0, searchsortedlast(counts, 8) - 1)
```

Comment:
Can you explain what's going on here? Might be helpful to define `8` as a variable or add a comment.

Reply:
I simplified this down and left a comment for what the rules of the function are in the docstr